### DISCUSSION TITLE
Showcases or how-to's for beginners?

### DISCUSSION THREAD

**Discussion by gacharles23** [adonisjs/core#4286] [Help]
I recently stumbled on AdonisJS and stepped through the Adocast series on "Let's Learn AdonisJS 5". The framework looks amazing - immediately hooking me! Unfortunately, there don't seem to be a lot of recent "how-to" guides for development through to a fully-deployed app. For people like me who don’t have deep expertise in production deployment, are there resources you can point me to for things like deploying to cloud-platforms like AWS and connecting to Amazon RDS (PostgreSQL) or DynamoDB? Ditto for testing: I watched a video from Aman Virk from 1.5 years ago (Exploring the AdonisJS tests runner) but it’s a bit too advanced for me. Are there any guides that are a bit more targeted for beginners? The AdonisJS framework looks wonderful. I hope I can use it. Thanks. -Grace

**Comment by GabrielNBDS** [comment]
Adonis is just a node app, the deploy process don't differ from other node apps so any "how to deploy node" should work. There is also [Cleavr](https://cleavr.io/) that specially handles Adonis deployment.
